Unity/Assets/YarnSpinner/Examples/Space/Space.unity

```diff
@@ -1894,6 +1899,31 @@ PrefabInstance:
       propertyPath: onDialogueEnd.m_PersistentCalls.m_Calls.Array.data[0].m_Arguments.m_BoolArgument
       value: 1
       objectReference: {fileID: 0}
+    - target: {fileID: 5935829475398715334, guid: 4ec3df5875e7347beb6176b545b8fb39,
```

Comment:
When loading a VM State that is supposed to show options, the UIBehaviour would not hide continue button, so I have added disabling OnOptionsStart call